test_simple_graph.py

```diff
@@ -11,6 +13,32 @@ def populated():
     return g
 
 
+@pytest.fixture
+def cyclic():
+    g = Graph()
+    g.add_node(1)
+    g.add_node(2)
+    g.add_node(3)
+    g.add_node(4)
+    g.add_node(5)
+    g.add_node(6)
+    g.add_node(7)
+    g.add_node(8)
+    g.add_node(9)
+    g.add_edge(1, 2)
+    g.add_edge(1, 3)
+    g.add_edge(1, 6)
+    g.add_edge(5, 1)
+    g.add_edge(2, 8)
+    g.add_edge(2, 9)
+    g.add_edge(2, 4)
+    g.add_edge(2, 3)
+    g.add_edge(3, 7)
+    g.add_edge(7, 1)
+    g.add_edge(7, 6)
+    return g
+
+
 def test_graph_constructor():
     assert Graph().graph == {}
 
```

Comment:
Personally, I would prefer that you check is Graph() is an instance of the Graph class, because if you change your implementation that test will always test that your graph is getting made.